src/dist/replication/meta_server/meta_bulk_load_service.cpp

```diff
@@ -16,21 +16,19 @@ bulk_load_service::bulk_load_service(meta_service *meta_svc, const std::string &
     _state = _meta_svc->get_server_state();
 }
 
-// ThreadPool: THREAD_POOL_META_STATE
+// ThreadPool: THREAD_POOL_META_SERVER
 void bulk_load_service::initialize_bulk_load_service()
 {
     task_tracker tracker;
-    error_code err = ERR_OK;
+    _sync_bulk_load_storage.reset(new mss::meta_storage(_meta_svc->get_remote_storage(), &tracker));
```

Comment:
Do you intend to set `_sync_bulk_load_storage` to a temporary tracker?

Reply:
Yes, `_sync_bulk_load_storage` will only be used in functions for sync bulk load info from remote storage, other operations still use `_meta_svc->get_meta_storage()` and track by meta service tracker.

Reply:
OK. Then I suggest making it a local variable, since as you implied, this variable only use in this function (`initialize_bulk_load_service`).

Reply:
It will be used in functions for sync bulk load info, you can see function `create_bulk_load_root_dir`, it will also be used in `sync_apps_bulk_load_from_remote_storage`. If I use it as a local variable, it should be pass to those functions.

Reply:
I remove `_sync_bulk_load_storage` in this pull request, I will update it in next pull request.